docs/1.8.9/chattrigers.md

```diff
@@ -0,0 +1,191 @@
+* Custom End Info - ?
+* Run Overview Overlay - [Sychic & Lily's Skytils](https://github.com/Skytils/SkytilsMod/releases/latest)
+* Run Splits Overlay - [Sychic & Lily's Skytils](https://github.com/Skytils/SkytilsMod/releases/latest)
+* Spirit Leap Names - [odtheking's Odin(Legit Version)](https://github.com/odtheking/Odin/releases/latest)
+* Blaze Timer - ?
+* Player Logging - ?
+* RNG Meter Display - ?
+* Terminal Splits - [odtheking's Odin(Legit Version)](https://github.com/odtheking/Odin/releases/latest)
+* Terminal Tracker - [odtheking's Odin(Legit Version)](https://github.com/odtheking/Odin/releases/latest)
+* Terminal Timer - [odtheking's Odin(Legit Version)](https://github.com/odtheking/Odin/releases/latest)
+* Hide Soulweaver Skulls - ?
+* Show Secret Clicks - [hannibal2's SkyHanni](https://modrinth.com/mod/skyhanni)
+* Dungeon Chest Profit - [NotEnoughUpdates](https://modrinth.com/mod/notenoughupdates)
+
+#### Solvers
+
+* Livid Solver - [Sychic & Lily's Skytils](https://github.com/Skytils/SkytilsMod/releases/latest)
+* Blaze Solver - [Sychic & Lily's Skytils](https://github.com/Skytils/SkytilsMod/releases/latest)
+* Quiz Solver - [Sychic & Lily's Skytils](https://github.com/Skytils/SkytilsMod/releases/latest)
+* Teleport Maze Solver - [Sychic & Lily's Skytils](https://github.com/Skytils/SkytilsMod/releases/latest)
+* Simon Says Solver - [Sychic & Lily's Skytils](https://github.com/Skytils/SkytilsMod/releases/latest)
+* Three Weirdos Solver - [Sychic & Lily's Skytils](https://github.com/Skytils/SkytilsMod/releases/latest)
+
+#### GUI
+
+* Party List Overlay - ?
+* Crystal Hollows Map - [Sychic & Lily's Skytils](https://github.com/Skytils/SkytilsMod/releases/latest)
+* Cells Align Timer - ?
+* Container Value - [hannibal2's SkyHanni](https://modrinth.com/mod/skyhanni)
+
+#### Party Finder
+
+* Auto Kick - [Sychic & Lily's Skytils](https://github.com/Skytils/SkytilsMod/releases/latest)
+* Better Party Finder Message - [Sychic & Lily's Skytils](https://github.com/Skytils/SkytilsMod/releases/latest)
+
+### NwjnAddons
+
+#### General
+
+* Draw Chat Waypoints - ?
+* Waypoint Color - ?
+* Chat Type Waypoint Rendering - ?
+* Waypoint Beacon Height - ?
+* Party Commands - ?
+* ➤ Leader Commands - ?
+* Rend Arrows - ?
+* Detect Reaper Armor - ?
+* Stops Entity Death Animation - ?
+* Skyblock XP Gain Message - ?
+* Treecap Timer - ?
+
+#### Bestiary
+
+* Mob Highlight - ?
+* Mob Highlight Color - ?
+* Armor Stand Names Highlight - ?
+* Armor Stand Highlight Color - ?
+* Player Highlight - ?
+* Player Highlight Color - ?
+* Matcho - ?
+* Arachnes Keeper - ?
+
+#### Crimson Isle
+
+* Trophy Fish Session Counter - ?
+* Announce Vanquishers - ?
+* Announce Jawbus - ?
+* Announce Thunder - ?
+* Announce Plhlegblasts - ?
+* Better Magma Boss Message - ?
+
+#### Mining
+
+* Mineshaft Waypoints - ?
+* Mineshaft Pity - ?
+
+#### QOL
+
+* Toggle Block Highlight - [Aycy's BlockOverlay](https://skyclient-files.pages.dev/Block_Overlay_4.0.3.jar)
```

Comment:
Link to the forum thread?

Reply:
no, forums thread leads to a mediafire link

Reply:
I would just link to a direct download. That URL is controlled by us anymore and isn’t used by SkyClient.

Reply:
direct dl from skyclient gh repo? it's fine to leave this for now, i can do post pull request across all pages for all projects via search and replace. not a blocking issue for this pr

Reply:
Yeah

Reply:
When #173 gets merged I will update the link here